lxd/instance/drivers/driver_qemu.go

```diff
@@ -1925,16 +1925,19 @@ func (d *qemu) architectureSupportsUEFI(arch int) bool {
 	return shared.ValueInSlice(arch, []int{osarch.ARCH_64BIT_INTEL_X86, osarch.ARCH_64BIT_ARMV8_LITTLE_ENDIAN})
 }
 
-func (d *qemu) setupNvram() error {
+func (d *qemu) setupNvram(skipVolMount bool) error {
```

Comment:
Could we use something like `setupNvram(mountVol bool)`, as it reads better when called `setupNvram(true)`.

Reply:
Thanks, fixed!